app-accessibility/edbrowse/edbrowse-3.7.5.1-r1.ebuild

```diff
@@ -15,24 +16,29 @@ IUSE="odbc"
 
 RDEPEND="
 	app-text/tidy-html5
-	>=net-misc/curl-7.36.0
-	>=dev-libs/libpcre-7.8
-	>=sys-libs/readline-6.0
 	dev-lang/duktape:=
+	dev-libs/libpcre
+	net-misc/curl
+	sys-libs/readline:=
 	odbc? ( dev-db/unixODBC )"
 DEPEND="${RDEPEND}"
-BDEPEND="${RDEPEND}
+BDEPEND="
 	dev-lang/perl
 	virtual/pkgconfig"
 
+PATCHES=(
+	"${FILESDIR}"/${P}-fno-common.patch
+	"${FILESDIR}"/${P}-manpage.patch
+)
+
 src_prepare() {
-	sed -i -e "s:/usr/share/doc/edbrowse:/usr/share/doc/${P}:" CMakeLists.txt
-	cmake-utils_src_prepare
+	sed -i -e "s:/usr/share/doc/edbrowse:/usr/share/doc/${PF}:" CMakeLists.txt || die
```

Comment:
in general, it's better to put the sed after the phase function

Reply:
Thanks, done